twitter/twitter.php

```diff
@@ -1288,7 +1288,7 @@ function twitter_fetchtimeline(int $uid): void
 	$osecret = DI::pConfig()->get($uid, 'twitter', 'oauthsecret');
 	$lastid  = DI::pConfig()->get($uid, 'twitter', 'lastid');
 
-	$application_name = DI::config()->get('twitter', 'application_name');
+	$application_name = DI::keyValue()->get('twitter_application_name');
```

Comment:
```suggestion
	$application_name = DI::keyValue()->get('twitter_application_name') ?? '';
```

Reply:
I think that the behaviour of these two classes is identical for a non found value.

Reply:
Yes, it returns `null`, which is incorrect since we use the return value as a string.